src/logic/chatGPT.js

```diff
@@ -0,0 +1,45 @@
+const axios = require("axios");
+import { key } from '../../key';
+
+
+export let chatResponse = '';
+
+export const createOpenAIAxiosClient = () => {
```

Comment:
do you need to export if you use this among this file? If you don't know what exactly i mean just read about export keyword. TL:DR you only use export when you want to use improt in another file to use the function there. For example like the key from key.js. 

Otherwise It's more safe to stick with no export const as it's hard to access and the code is more tight/hermetic. It's safer to use as JS won't let anyone access this in easy way.